Check application status (#45)
vippsas/vipps-partner: README.md

**Please note:** Vipps can not share information about the status of a
merchant's agreement or product orders.

> This is because we need explicit consent from the merchant to share this

Author: ```suggestion
This is because we need explicit consent from the merchant to share the
```

> The reason for this is that we need explicit consent from the merchant to share this

Author: ```suggestion
This is because we need explicit consent from the merchant to share this
```
